lightly/loss/directclr_loss.py

```diff
@@ -0,0 +1,91 @@
+""" Contrastive Loss Functions """
+
+# Copyright (c) 2020. Lightly AG and its affiliates.
+# All Rights Reserved
+
+
+from typing import Any
+
+from torch import Tensor
+
+from lightly.loss import NTXentLoss
+
+
+class DirectCLRLoss(NTXentLoss):
+    """Implementation of the NT-Xent based DirectCLR Loss.
+
+    Following the DirectCRL[0] paper, this loss should be used without projection
+    head. Set loss_dim to the desired truncated representation length.
```

Comment:
Nit:
```suggestion
    head. Set `loss_dim` to the desired truncated representation length.
```
This will render nicely once we add it to the docs. Same thing further down.